spring-boot-project/spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -4377,6 +4377,29 @@ in your configuration, e.g. `spring.data.neo4j.uri=file://var/tmp/graph.db`.
 
 
 
+[[boot-features-neo4j-ogm-native-types]]
+==== Using native types
+Neo4j-OGM can map some types, like `java.time.*` to `String` based properties or
+use one of the various native types that Neo4j provide
+(Please refer to the https://neo4j.com/docs/developer-manual/3.4/drivers/cypher-values/#driver-neo4j-type-system[Cypher type system] for a full list).
```

Comment:
There is a hard-coded version there and we'd prefer not to have to chase those links down. There are other versions used in the doc already, see `index.adoc` for examples.